src/android/Bugsnag.cpp

```diff
@@ -229,35 +222,40 @@ JNIEXPORT void JNICALL Java_com_bugsnag_android_BugsnagCocos2dxPlugin_configureN
       bugsnag::HashMapClass, "put",
       "(Ljava/lang/Object;Ljava/lang/Object;)Ljava/lang/Object;");
 
+  bugsnag::BreadcrumbTypeClass = env->FindClass("com/bugsnag/android/BreadcrumbType");
+  _mapBreadcrumbType(env, bugsnag::Navigation, "ERROR");
+  _mapBreadcrumbType(env, bugsnag::Request, "REQUEST");
+  _mapBreadcrumbType(env, bugsnag::Process, "PROCESS");
+  _mapBreadcrumbType(env, bugsnag::Log, "LOG");
+  _mapBreadcrumbType(env, bugsnag::User, "USER");
+  _mapBreadcrumbType(env, bugsnag::State, "STATE");
+  _mapBreadcrumbType(env, bugsnag::Error, "ERROR");
+  _mapBreadcrumbType(env, bugsnag::Manual, "MANUAL");
+
   jclass bugsnag = env->FindClass("com/bugsnag/android/Bugsnag");
   bugsnag::BugsnagClass = (jclass)env->NewGlobalRef(bugsnag);
   env->DeleteLocalRef(bugsnag);
   bugsnag::BugsnagResumeSession = env->GetStaticMethodID(
```

Comment:
While we didn't do it previously, should these not be GlobalRef'ed?

Reply:
They stay valid so long as the class ref stays valid (which it does by virtue of the above `NewGlobalRef`)